src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -179,6 +209,8 @@ internal static Summary[] Run(BenchmarkRunInfo[] benchmarkRunInfos)
                 logger.WriteLineInfo($"//   {benchmark.DisplayInfo}");
             logger.WriteLine();
 
+            var eventHandler = config.GetCompositeEventHandler();
```

Comment:
could you please pass an instance of composite handler from the calling method here (we already do that for logger)?